app/src/pages/playground/PlaygroundExperimentRunDetailsDialog.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useMemo } from "react";
+import { graphql, useLazyLoadQuery } from "react-relay";
+
+import {
+  Card,
+  CardProps,
+  Dialog,
+  DialogContainer,
+  Flex,
+  View,
+} from "@arizeai/components";
+
+import { CopyToClipboardButton } from "@phoenix/components";
+import { JSONBlock } from "@phoenix/components/code";
+
+import type { PlaygroundExperimentRunDetailsDialogQuery } from "./__generated__/PlaygroundExperimentRunDetailsDialogQuery.graphql";
+
+/**
+ * A slide-over that shows the details of a playground experiment run.
+ */
+export function PlaygroundExperimentRunDetailsDialog({
+  runId,
+}: {
+  runId: string;
+}) {
+  const data = useLazyLoadQuery<PlaygroundExperimentRunDetailsDialogQuery>(
+    graphql`
+      query PlaygroundExperimentRunDetailsDialogQuery($runId: GlobalID!) {
+        run: node(id: $runId) {
+          ... on ExperimentRun {
+            output
+            example {
+              id
+              revision {
+                input
+                output
+              }
+            }
+          }
+        }
+      }
+    `,
+    { runId }
+  );
+  const run = useMemo(() => {
+    const run = data.run;
+    const example = run.example;
+    const revision = example?.revision;
+    return {
+      exampleId: example?.id,
+      input: JSON.stringify(revision?.input, null, 2),
+      referenceOutput: JSON.stringify(revision?.output, null, 2),
+      runOutput:
+        run.output !== null ? JSON.stringify(run.output, null, 2) : null,
+    };
+  }, [data]);
+  const { exampleId, input, referenceOutput, runOutput } = run;
+  return (
+    <DialogContainer type="slideOver" isDismissable onDismiss={() => {}}>
```

Comment:
Clicking anywhere within the dialog closes it, including when trying to collapse cards or select text


https://github.com/user-attachments/assets/f3ec3ecf-3ea4-4d91-8a7c-00d4a6cbe575